src/components/SeaBattle/src/Field.js

```diff
@@ -0,0 +1,120 @@
+
+export default class Field {
+
+    constructor() {
+        this.field =new Array(10).fill()
+            .map( el => (new Array(10).fill("*")));
+        this.shipCounter = 0;
+    }
+
+    get getField() {
+        return this.field;
+    }
+
```

Comment:
suggestion: you use `.getField[row][col]` so it would be great if you added some method, like:
```
getCell(row, col) {
  return this.field[row][col];
}
```